### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by fjglira — APPROVED** [review]

File: common/scripts/kind_provisioner.sh [hunk 1]
```diff
@@ -196,7 +196,7 @@ EOF
     return 9
   fi
   # Workaround kind issue causing taints to not be removed in 1.24
-  kubectl taint nodes "${NAME}"-control-plane node-role.kubernetes.io/control-plane- || true
+  kubectl taint nodes "${NAME}"-control-plane node-role.kubernetes.io/control-plane- 2>/dev/null || true
```
Comment (fjglira) [inline common/scripts/kind_provisioner.sh]: This file is the same one that the upstream test has, maybe we should refer directly to that file and make the change there?

Comment (luksa) [reply common/scripts/kind_provisioner.sh]: Argh. Thanks.

**Pull request merged**